### PR TITLE
[FEATURE] Criar component para o card de Trilha

### PR DESCRIPTION
**Pull request by mathvalenza**
This is the first version of the Trail component, that shows the title, an image and a abstract of the trail

### CODE REVIEW

File: src/views/Trail.vue [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<template>
+<div>
```
Comment (rartner) [inline src/views/Trail.vue]: missing indent?

File: src/views/Trail.vue [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    imageSrc: {
+      type: String,
+      default: '',
+    } 
+  }
+}
+</script>
+
+<style scoped>
+
```
Comment (rartner) [inline src/views/Trail.vue]: has no content. couldn't be removed?

File: src/views/Trail.vue [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+</style>
```
Comment (cuzik) [inline src/views/Trail.vue]: Missing extra line on the end.

File: src/views/Home.vue [hunk 4]
```diff
@@ -1,35 +1,25 @@
...
-<!-- Add "scoped" attribute to limit CSS to this component only -->
+<script>
+import Trail from './Trail';
+
+export default {
+  name: 'home',
+  components: {
+    Trail,
+  },
+};
```
Comment (cuzik) [inline src/views/Home.vue:19]: Remove `;`

File: src/views/Home.vue [hunk 5]
```diff
@@ -1,35 +1,25 @@
...
+      title="Trilha 1"
+      abstract="Esta é a primeira trilha de todas. Vamos lá?"
+      image-src=""
+    />
   </v-container>
 </template>
 
-<!-- Add "scoped" attribute to limit CSS to this component only -->
+<script>
+import Trail from './Trail';
```
Comment (cuzik) [inline src/views/Home.vue]: Remove ;

**Review by cuzik — CHANGES_REQUESTED** [review]
Can we write test for these components?

Resolve the conflicts.

**Comment by cuzik** [comment]
[[FEATURE] Criar component para o card de Trilha (3)](https://trello.com/c/8PhZcoMp/63-feature-criar-component-para-o-card-de-trilha-3)

File: tests/unit/Home.spec.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import { mount } from '@vue/test-utils';
+import Home from '../../src/views/Home.vue';
```
Comment (rartner) [inline tests/unit/Home.spec.js]: `@/views/Home.vue` should work here

File: tests/unit/Login.spec.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import { mount } from '@vue/test-utils';
+import Login from '../../src/views/Login.vue';
```
Comment (rartner) [inline tests/unit/Login.spec.js]: `@/views/Login.vue` should work here

File: tests/unit/Trail.spec.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+import { mount } from '@vue/test-utils';
+import Trail from '../../src/views/Trail.vue';
```
Comment (rartner) [inline tests/unit/Trail.spec.js]: `@/views/Trail.vue` should work here

Comment (rartner) [reply tests/unit/Trail.spec.js]: I think `Trail` is a `component`, not a `view`

**Review by rartner — APPROVED** [review]

**Review by cuzik — APPROVED** [review]
:cow:

**Pull request merged**